nautobot/extras/datasources/git.py

```diff
@@ -71,7 +71,9 @@ def enqueue_pull_git_repository_and_refresh_data(repository, request):
 
 
 def get_job_result_and_repository_record(repository_pk, job_result_pk, logger):
-    """Get JobResult Instance and GItRepository Instance"""
+    """Get JobResult instance and GitRepository instance"""
+    GitJobResult = namedtuple("GitJobResult", ["job_result", "repository_record"])
```

Comment:
This is nearly perfect!

Can you please:

- [ ] Move `GitJobResult` to a module-level global so that it didn't redefined every time this function is called, and so it may also be reusable?
- [ ] Please also add a comment describing its use 
- [ ] Update the docstring for `get_job_result_and_repository_record` to indicate it returns a `GitJobResult`?

My apologies for not being more specific!